tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -338,30 +336,6 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
       ]
     },
     {
-      code: '<App>{\'foo "bar"\'}</App>',
-      output: '<App>foo \"bar\"</App>',
```

Comment:
This autofix could have been kept. `<App>foo "bar"</App>` is valid.

Reply:
I don't remember – have we addressed or talked about this one?

Reply:
@lydell 

Yeah, since it is valid, I added back the fix for cases like this and the tests.